### DISCUSSION TITLE
[rocm-llvm] Build fails at install step

### DISCUSSION THREAD

**Discussion by kozross** [rocm-arch/rocm-arch#706] [Support] [answered]
The build step works fine, but then at the install step, I get an error like this:

```
-- Installing: /home/koz/.cache/paru/clone/rocm-llvm/pkg/rocm-llvm/opt/rocm/llvm/bin/verify-uselistorder
CMake Error at docs/cmake_install.cmake:46 (file):
  file INSTALL cannot find
  "/home/koz/.cache/paru/clone/rocm-llvm/src/docs/ocamldoc/html/.": No such
  file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:85 (include)


FAILED: CMakeFiles/install.util 
cd /home/koz/.cache/paru/clone/rocm-llvm/src && /usr/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in package().
    Aborting...
error: failed to build 'rocm-llvm-5.0.2-2': 
error: packages failed to build: rocm-llvm-5.0.2-2
```

**Comment by acxz** [comment, marked as answer]
Can you confirm that you are building in a clean chroot?

For an easy way to build in a clean chroot take a look at paru's `--chroot` documenation.

**Comment by kozross** [comment]
A clean chroot build works fine. This is very strange - I've never done this before, and never saw issues like this.

**Comment by j-stephan** [comment]
I am experiencing the same issue. I believe that this is related to OCaml being installed on the build system which is picked up by LLVM and might lead to this problem.

**Reply by acxz** [reply]
If you find a fix for this, feel free to make a PR. However, this is not that big of a priority for us since we recommend/require users to build in clean chroot.
